### PR TITLE
Preparing a new tag without the conflicting operation "Operations_List"

### PR DESCRIPTION
**Pull request by tokaplan**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Add a changelog entry for this PR by answering the following questions:
  1. What's the purpose of the update?
      - [ ] new service onboarding
      - [ ] new API version
      - [ ] update existing version for new feature
      - [x] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify
  2. When are you targeting to deploy the new service/feature to public regions? Please provide the date or, if the date is not yet available, the month.
  3. When do you expect to publish the swagger? Please provide date or, the the date is not yet available, the month.
  4. If updating an existing version, please select the specific langauge SDKs and CLIs that must be refreshed after the swagger is published.
      - [x] SDK of .NET (need service team to ensure code readiness)
      - [x] SDK of Python
      - [x] SDK of Java
      - [x] SDK of Js
      - [x] SDK of Go
      - [x] PowerShell
      - [x] CLI
      - [x] Terraform
      - [ ] No refresh required for updates in this PR

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist 

> **Applicability**: :warning: 
>
> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
> - Change to data plane APIs 
> - Adding new properties 
> - All removals

Otherwise your PR may be subject to ARM review requirements. Complete the following:
- [ ] Check this box if any of the following apply to the PR so that label "WaitForARMFeedback" will be added automatically to begin ARM API Review. Failure to comply may result in delays to the manifest.
  - Adding a new service
  - Adding new API(s)
  - Adding a new API version
    -[ ] To review changes efficiently, ensure you copy the existing version into the new directory structure for first commit and then push new changes, including version updates, in separate commits.

- [ ] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If any of the following scenarios apply to the PR, request approval from the Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 
- [ ] Removing API(s) in a stable version
- [ ] Removing properties in a stable version
- [ ] Removing API version(s) in a stable version
- [ ] Updating API in a stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Review by ArcturusZhang — COMMENTED** [review]
This LGTM since it resolves the issue. But actually I am expecting that we could introduce a new file that only contains the Operations_List operation for better maintenance.
Does this make sense to you?

File: specification/applicationinsights/resource-manager/readme.md [hunk 1]
```diff
@@ -222,6 +222,31 @@ directive:
...
+  - Microsoft.Insights/stable/2015-05-01/componentProactiveDetection_API.json
+  - Microsoft.Insights/stable/2015-05-01/componentWorkItemConfigs_API.json
+  - Microsoft.Insights/stable/2015-05-01/favorites_API.json
+  - Microsoft.Insights/stable/2015-05-01/webTestLocations_API.json
+  - Microsoft.Insights/stable/2015-05-01/webTests_API.json
+  - Microsoft.Insights/stable/2015-05-01/analyticsItems_API.json
+  - Microsoft.Insights/stable/2020-11-20/workbookTemplates_API.json
+  - Microsoft.Insights/stable/2021-03-08/myworkbooks_API.json
+  - Microsoft.Insights/stable/2021-08-01/workbooks_API.json
+  - Microsoft.Insights/preview/2018-05-01-preview/components_API.json
```
Comment (vutran01) [inline specification/applicationinsights/resource-manager/readme.md:245]: >2018-05-01-preview/components_API.json [](http://example.com/codeflow?start=31&length=38)

should components_api.json be from a stable version? [https://github.com/Azure/azure-rest-api-specs/blob/main/specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/components_API.json](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/components_API.json)

once when it's been released it should be from the /stable

Comment (tokaplan) [reply specification/applicationinsights/resource-manager/readme.md:245]: There is actually an even newer stable version _stable/2020-02-02_, using that one.

File: specification/applicationinsights/resource-manager/readme.md [hunk 2]
```diff
@@ -222,6 +222,31 @@ directive:
   
 ```
 
+### Tag: package-2022-01-11
```
Comment (vutran01) [inline specification/applicationinsights/resource-manager/readme.md:225]: >package-2022-01-11 [](http://example.com/codeflow?start=9&length=18)

we call every Tag "package-" but maybe we should have special tag if we plan to publish the sdk for it? either that or have appropriate comments so the next person doesn't overwrite or regress the versions. maybe all add links to "best practices" and other sdk architecture

• [https://github.com/Azure/azure-sdk-for-rust/issues/563](https://github.com/Azure/azure-sdk-for-rust/issues/563)
• [https://github.com/Azure/azure-sdk-for-rust/pull/567](https://github.com/Azure/azure-sdk-for-rust/pull/567)
Please also read these docs:
• [https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/481/API-Spec-SDK-Package-Versioning-Considerations](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/481/API-Spec-SDK-Package-Versioning-Considerations)
• [https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/83/Manual-Process-Adding-new-API-version](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/83/Manual-Process-Adding-new-API-version)

Comment (tokaplan) [reply specification/applicationinsights/resource-manager/readme.md:225]: @vutran01 , is there another reason to declare a tag? I thought they were only for generating SDKs... I'll add the links, that's a good idea I think.

Comment (vutran01) [reply specification/applicationinsights/resource-manager/readme.md:225]: yes, for building the sdk but not every tag is used to publish the sdk

Comment (tokaplan) [reply specification/applicationinsights/resource-manager/readme.md:235]: Added the links. Feel free to propose a convention, I don't think I have enough understanding of the overall process to have an opinion here. I don't even know why we would want to build an SDK and then not publish it.

**Review by vutran01 — APPROVED** [review]
:shipit:

**Comment by tokaplan** [comment]
@akning-ms , this PR is being marked as Breaking change. While that is probably technically correct, the change here is to remove a duplicate definition of an operation that is already defined in another file that is a part of the same package. Therefore, there shouldn't be any impact on the resulting API surface. Can you please take a look and help me figure out a way to unblock this PR and proceed? Thank you.

**Comment by akning-ms** [comment]
> @akning-ms , this PR is being marked as Breaking change. While that is probably technically correct, the change here is to remove a duplicate definition of an operation that is already defined in another file that is a part of the same package. Therefore, there shouldn't be any impact on the resulting API surface. Can you please take a look and help me figure out a way to unblock this PR and proceed? Thank you.

If it is just removing duplicate definition, it is ok for the breaking change,  not sure where is another operationList definition, but Is it make sense for you as @ArcturusZhang 's comment to move it in a separate file?

**Comment by tokaplan** [comment]
> > @akning-ms , this PR is being marked as Breaking change. While that is probably technically correct, the change here is to remove a duplicate definition of an operation that is already defined in another file that is a part of the same package. Therefore, there shouldn't be any impact on the resulting API surface. Can you please take a look and help me figure out a way to unblock this PR and proceed? Thank you.
> 
> If it is just removing duplicate definition, it is ok for the breaking change, not sure where is another operationList definition, but Is it make sense for you as @ArcturusZhang 's comment to move it in a separate file?

The other definition is here in [aiOperations_API.json](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/aiOperations_API.json#L37). It's already a separate file.

**Pull request merged**